shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/mysql/visitor/statement/impl/MySQLDALStatementSQLVisitor.java

```diff
@@ -974,13 +974,15 @@ private VariableAssignSegment getVariableAssign(final OptionValueContext ctx) {
     
     @Override
     public ASTNode visitSetCharacter(final SetCharacterContext ctx) {
-        MySQLSetStatement result = new MySQLSetStatement();
+        final MySQLSetStatement result = new MySQLSetStatement();
```

Comment:
@soulasuna Please remove final modifier here.

Reply:
This final for `<module name="VariableDeclarationUsageDistance"/>` can not removed.